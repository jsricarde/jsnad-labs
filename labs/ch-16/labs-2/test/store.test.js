'use strict'
const store = require('../store')

test('handles network errors', (done) => {
  store(null, (err) => {
    expect(err).toStrictEqual(Error('input must be a buffer'))
    done()
  })
})

test('responds with data', (done) => {
  store(Buffer.from('holi'), (err, data) => {
    expect(err == null).toBe(true)
    //expect(Buffer.isBuffer(data)).toBeTruthy()
    expect(data).toMatchSnapshot()
    done()
  })
})