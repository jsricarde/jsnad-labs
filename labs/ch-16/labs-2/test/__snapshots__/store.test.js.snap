// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`responds with data 1`] = `
Object {
  "id": "kzoc",
}
`;
